drafts/2.4-representable-functors.md

```diff
@@ -95,14 +111,17 @@ class Representable f where
     index    :: f x -> Rep f -> x
 ```
 ```kotlin:ank:silent
-
+interface Representable<F, A> {
```

Comment:
fix the indents in your body and the type signutares are wrong. Representable is a Typeconstructor. It solely needs the `F`
```suggestion
interface Representable<F> {
```